veracruz-server-test/src/main.rs

```diff
@@ -702,10 +701,12 @@ mod tests {
     ) -> Result<(), VeracruzServerError> {
         info!("### Step 0.  Initialise test configuration.");
         // initialise the pipe
+        #[allow(clippy::type_complexity)]
```

Comment:
Should we have a global clippy configuration instead of using attributes? Cf. https://github.com/rust-lang/rust-clippy/blob/master/README.md#configuration

Reply:
Yes, I think this is a good idea.